Tensile/KernelWriterAssembly.py

```diff
@@ -1698,7 +1728,7 @@ def initKernel(self, kernel, tPA, tPB ):
 
     self.defineSgpr("OrigLoopCounter", 1)
 
-    if self.prefetchAcrossPersistent0:
+    if self.prefetchAcrossPersistent0 and (not self.noTailLoop):
```

Comment:
This is to improve sgpr allocation for StoreCInUnroll.
StoreCInUnroll requires noTailLoop optimization and TailLoopCounter is not used at all in that case.